lib/chains.ts

```diff
@@ -154,7 +154,6 @@ export const CHAIN_INFO = {
     networkType: NetworkType.L1,
     l1: chain.mainnet,
     explorer: "https://etherscan.io/",
-    explorerAPI: "https://api.etherscan.io/api",
```

Comment:
> Etherscan API V1 will be fully deprecated by August 15, 2025. 

So it's already deprecated and `getBlocksFromTimestamps()` and `getBlockByNumber()` aren't used anywhere anyway https://github.com/livepeer/explorer/issues/304